src/components/Films/Films.tsx

```diff
@@ -25,21 +27,41 @@ export interface IItems {
 
 const Films = () => {
   const data = useSelector((state: RootState) => state.film);
-  console.log(data);
+  const { Option } = Select;
   const [title, setTitle] = useState('');
+  const [page, setPage] = useState<number>(1);
+
+  const handleChange = (value: number) => {
+    setPage(value);
+  };
+
   const dispatch = useDispatch();
   const getFilmInfo = () => {
     dispatch(getFilmData(title));
   };
+
   useEffect(() => {
     getFilmInfo();
   }, [title]);
+
   return (
     <div className="films-root">
       <div className="search-sect">
-        <label>Search</label>
-        <input type="text" value={title} onChange={(e) => setTitle(e.target.value)} />
-        <button>Search</button>
+        <Input
+          type="text"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+          className="filmSearch"
+          placeholder="Film Name"
+        />
+        <Button type="primary" shape="circle" icon={<SearchOutlined />} />
+        <Select value={page} style={{ width: 120 }} className="selectMain" onChange={handleChange}>
```

Comment:
why ????????? why are you customly give number of pages ? it should debend on items coming from API and items per page you define